### DISCUSSION TITLE
can’t user IHandle<Dictionary<string, object>>

### DISCUSSION THREAD

**Discussion by wuhl508159280** [canton7/Stylet#240] [Q&A]
I have mainWindown,Usercontrol is in the ContentControl container in mainWindon。
`   public void Handle(Dictionary<string, object> message)
        {
            if (message is null) return;
            foreach (var v in message)
            {
                Cnds.Common.LocalLoggingService.Info("value：" + v.Value.ToString());
                switch (v.Key)
                {`
                      case "Content":
                        Main.Content = v.Value;
                        break;
               }
            }
but usercontrol also have  IHandle<Dictionary<string, object>>
this error,because i don't  send message for usercontrol by Token

**Comment by canton7** [comment]
I don't understand what your question is I'm afraid.
